slsDetectorSoftware/src/slsDetector.cpp

```diff
@@ -1253,6 +1279,29 @@ void slsDetector::setNumberOfTriggers(int64_t value) {
     shm()->nTriggers = value;
     sendTotalNumFramestoReceiver();
 }
+
+int64_t slsDetector::getNumberOfBurstsFromShm() {
```

Comment:
Is this function used? When searching I don't find it...

Reply:
I missed the progress by multi. Removed all the getNumberofFramesfromShm, triggersfromshm, and burstsfromshm.. and made into one function to get getTotalNumFramestoReceive.